components/automate-workflow-server/habitat/default.toml

```diff
@@ -8,8 +8,8 @@
 api_port = 9611
 audit_max_events = 100
 ca_cert_chain_depth = 2
-db_pool_init_count = 20
-db_pool_max_count = 100
+db_pool_init_count = 10
+db_pool_max_count = 20
```

Comment:
I guess we never made these ones actually configurable, which is fine because no on uses this component right now.